.travis.yml

```diff
@@ -22,10 +25,24 @@ jobs:
       name: "OpenJDK 8"
       jdk: openjdk8
       script: mvn clean verify -B
+    - stage: test
+      arch: arm64
+      name: "OpenJDK 8"
+      dist: bionic
+      addons:
+        apt:
+          - openjdk-8-jdk
+      jdk: openjdk8
+      script: mvn clean verify -B
     - name: "OpenJDK 11"
       jdk: openjdk11
       script: mvn clean verify -B
-    # GraalVM
+    - name: "OpenJDK 11"
```

Comment:
Can we enable this only on master?

Reply:
I have enabled the openjdk arm64 tests for master only